js/components/cohortbuilder/components/VisitOccurrenceTemplate.html

```diff
@@ -101,7 +101,7 @@
 					</div>
 				</td>
 			</tr>
-			<tr data-bind="if: PlaceOfServiceLocation() !== undefined, visible: PlaceOfServiceLocation() !== undefined">
+			<tr data-bind="if: PlaceOfServiceLocation() !== null, visible: PlaceOfServiceLocation() !== null">
```

Comment:
could you try just:
```
if: PlaceOfServiceLocation, visible: PlaceOfServiceLocation
```

The bindings will automatically unwrap for you, and it will check the 'falsy' value of the binding. that way you don't have any confusion between undefiend or null, since those are both falsy.

Reply:
@chrisknoll i just tried, it doesn't work

Reply:
@chrisknoll , the issue here is in data supplied from WebAPI, UI is not the place to fix it. Previously, WebAPI didn't return nulls for empty filters, but later @aklochkova unintentionally changed that to do so. Therefore, the distinction between undefined and null was lost